src/chromo/models/dpmjetIII.py

```diff
@@ -146,6 +140,23 @@ def _cross_section(self, kin=None, precision=None):
 
         # TODO set more cross-sections
 
+    @property
+    def hA_AA_glauber_trials(self):
+        return self._lib.dtglgp.jstatb
+
+    @hA_AA_glauber_trials.setter
+    def hA_AA_glauber_trials(self, ntrials):
+        """Set number of trials for Glauber model integration.
+
+        Larger number of `ntrials` reduces the fluctuations in the cross section,
+        thus, making it more smooth. Smaller number of `ntrials` makes calculations of
+        cross section faster.
+
+        Args:
+            ntrials: number of trials. Default is 1000 (set at model initialisation).
```

Comment:
There are no args in this case. The comment for the property should go into the getter. Instead of "Get/set X" the brief description should say "Access X"

Reply:
> Note the use _ for the irrelevant name of the setter.

I tried, it doesn't work. Although I thought that this should work.

The long technical name `hA_AA_glauber_trials` was intended to "scare off" users that don't need it. But I guess, if we would really wanted to hide it, then we would make this property private. So, indeed, if it is public, then the simple name is better.

Reply:
> I tried, it doesn't work. Although I thought that this should work.

Ok, you are right, I am sorry that I remembered that wrong. It is really surprising.